boot.php

```diff
@@ -144,11 +144,11 @@
 			$jsCode[] = '  redactorInit();';
 			$jsCode[] = '});';
 
-			if (!rex_file::put($this->getAssetsUrl('cache/redactor2_profiles.js').'', implode(PHP_EOL, $jsCode))) {
+			if (!rex_file::put($this->getAssetsPath('cache/redactor2_profiles.js').'', implode(PHP_EOL, $jsCode))) {
```

Comment:
@phoebusryan 
Warum legst du die Datei nicht im eigentlichen Cache Ordner 
`getCachePath() > /redaxo/cache/addons/redactor2/` ab?

Bis jetzt liegt diese hier `/assets/addons/redactor2/cache/`

Reply:
Historisch; ich wusste es damals nicht besser. Deshalb liegen auch die redactor2.js-Vendor Files nicht im Vendorfolder. Muss ich irgendwann mal nachholen :)

Reply:
> Muss ich irgendwann mal nachholen

Bis dahin schreibt der PR die Daten korrekt in dein gewünschtes Verzeichnis. getAssetsUrl ist relative und kann wie bei mir heute, am falschen Ort landen.